api_app/analyzers_manager/observable_analyzers/ailtyposquatting.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import math
+
+from ail_typo_squatting import runAll
+from ail_typo_squatting.dns_local.resolving import dnsResolving
+
+from api_app.analyzers_manager import classes
+
+logger = logging.getLogger(__name__)
+
+
+class AilTypoSquatting(classes.ObservableAnalyzer):
+    """
+    wrapper for https://github.com/typosquatter/ail-typo-squatting
+    """
+
+    dns_resolving: bool = False
+
+    def update(self) -> bool:
+        pass
+
+    def run(self):
+        response = {}
+        logger.info(
+            f"""running AilTypoSquatting on {self.observable_name}
+                    with tlp {self._job.tlp}
+                    and dns resolving {self.dns_resolving}"""
+        )
+        resultList = []
+        response["algorithms"] = runAll(
```

Comment:
can you please show the results from this? cause "formatoutput"="yara" I don't think it is the right option

Reply:
![image](https://github.com/intelowlproject/IntelOwl/assets/102874321/23f30a75-8a9a-46c3-876b-4ee69c2ef885)

Reply:
yeah I mean it does not make sense to me cause I should have expected "yara" code output. But ok, nevermind, the important thing is that it works. Maybe the written file would be yara but we are not generating it.
So, it could make sense to set it as None too to avoid confusion for the next reader

Reply:
says : `known format: None. Will use text format instead`
committing with `text` only.